tycho-core/src/main/java/org/eclipse/tycho/core/ee/EEVersion.java

```diff
@@ -15,8 +15,6 @@
 
 import java.util.Objects;
 
-import javax.annotation.Nonnull;
```

Comment:
No longer using `javax.annotation.Nonnull` is a good move, but removal looks like a step backwards, no?

Reply:
Given that (I haven counted them) there are many hundreds of file not suing it compared to only these few its jsut a baby-step :-)